js/deer-render.js

```diff
@@ -199,7 +199,76 @@ DEER.TEMPLATES.shadow = (obj, options = {}) => {
                     ${entries}
                     </div>`
                 })
-                .catch(err=>elem.innerHTML="Error loading external data.")
+                .catch(err => elem.innerHTML = "Error loading external data.")
+        }
+    }
+}
+
+DEER.TEMPLATES.transcriptionStatus = function (obj, options = {}) {
+    if (!obj['@id']) { return null }
+
+    return {
+        html: `
+        looking up transcription status... `,
+        then: (elem) => {
+            const query = {
+                target: obj['@id'],
+                "body.transcriptionStatus": { $exists: true }
+            }
+            fetch(DEER.URLS.QUERY, {
+                method: "POST",
+                headers: {
+                    "Content-Type": "application/json"
+                },
+                body: JSON.stringify(query)
+            }).then(response => response.json())
+                .then(data => {
+                    elem.dataset.transcriptionStatus = data[0]?.body.transcriptionStatus ?? "in progress"
+                    elem.innerHTML = elem.dataset.transcriptionStatus !== "in progress"
+                        ? `✔ Reviewed by <deer-view deer-id="${data[0].body.transcriptionStatus}" deer-template="label">${data[0].body.transcriptionStatus}</deer-view>`
+                        : `❌ Not yet reviewed (click to approve)`
+                    if (data.length) {
+                        elem.setAttribute(DEER.SOURCE, data[0]['@id'])
+                    }
+                    elem.classList[elem.dataset.transcriptionStatus !== "in progress" ? "add" : "remove"]("success")
+                    setTimeout(() => UTILS.broadcast(undefined, DEER.EVENTS.NEW_VIEW, document, elem.querySelector(DEER.VIEW)), 0)
```

Comment:
Getting an error on this broadcast on my localhost.  I think it is when no transcriptionStatus Annotation yet exists.  Once I make one, whether active or inactive, I no longer get the error. 
deer-utils.js:358 Uncaught TypeError: Cannot convert undefined or null to object
    at Function.assign (<anonymous>)
    at Object.broadcast (deer-utils.js:358:56)
    at deer-render.js:234:44